help/en/html/doc/Technical/ReleaseProcess.md

```diff
@@ -0,0 +1,227 @@
+# (Proposed) JMRI Release Methodology
+
+On the jmri-developers list, it's been proposed to move toward a release process aligned with [semantic versioning](https://semver.org) in which an I.J.K release number indicates the kinds of included changes:
+
+ - Changes that can _break_ outside code cause the first number (I) to be incremented and the other to reset to zero.
+ - Other changes that appear in the visible API will increment the 2nd number (J) and reset the third
+ - Other changes result in incrementing the third number (K) 
+ 
+## Denoting releases 
+
+JMRI installers can be built in four different scenarios:
+
+* Developer build on their local machine
+* Jenkins 'nightly' build as part of the Continuous Integration framework
+* Release build distributed as installers on the JMRI website
+* _Recommended_ installer builds distributed on the JMRI website and promoted as the main supported version until the next _recommended_ release
+
+Developer builds will identify themselves with versions strings like `JMRI-dev-major+private+jake+20200625T1452Z+Rbab33d0f33` where `dev-major` denotes the branch, `private` denotes a developers own private build, `jake` the local username of the specific developer, and `+R` indicates the 8 digits of the SHA of HEAD when built.
+
+Jenkins 'nightly' builds will identify themselves with version strings like `JMRI-dev-minor+CI+Jenkins+20200625T1452Z+Rbab33d0f33` where `dev-major` denotes the branch, `Jenkins` that this was built by the Jenkins CI environment, and `+R` indicates the 8 digits of the SHA of HEAD when built.
+
+A released version is denoted as a `vI.J.K` (i.e. `v21.4.2`) tag in our common repository.
+
+Distributed installers built from a released version will identify themselves with version strings like `JMRI.21.4.2+Rbab3d0f33` where the `+R` indicates 8 digits of the tag SHA.  
```

Comment:
"the 8 digits" should probably be "the first 8 digits"

Reply:
Fixed that (two places) in the doc, thanks.